Makefile

```diff
@@ -24,7 +24,8 @@ verify-format:
 .PHONY: publish-snapshots
 publish-snapshots:
 ifeq ($(CIRCLE_BRANCH),master)
-	./gradlew artifactoryPublish
+  # TODO(anuraaga): Remove version specificer after next release creates a tag on master.
+	./gradlew artifactoryPublish -Prelease.version=0.10.0-SNAPSHOT
```

Comment:
Cannot you just create a correct tag on the main branch for the current released version?

Reply:
Well, we already have a tag referring to a release branch which is what we built from. While moving it to the main branch could be effectively the same, I think it's some rewriting history that I want to avoid, this workaround isn't so bad I think.